src/Apps/Order2/Routes/Checkout/Order2CheckoutRoute.tsx

```diff
@@ -62,81 +81,421 @@ export const Order2CheckoutRoute: React.FC<Order2CheckoutRouteProps> = ({
       />
       <GridColumns>
         <Column span={[12, 8]} start={[1, 2]}>
-          <Box backgroundColor="mono0" py={2} px={2}>
-            <Flex justifyContent="space-between">
-              <Text flex={1} variant="xs">
-                Order Summary
-              </Text>
-              <ChevronDownIcon />
-            </Flex>
-          </Box>
           <Stack gap={1} bg="mono5">
-            <Flex flexDirection="column" backgroundColor="mono0" py={2} px={2}>
-              <Text variant="sm">Delivery address</Text>
-              <Formik
-                initialValues={{
-                  address: {
-                    name: "",
-                    country: "",
-                    postalCode: "",
-                    addressLine1: "",
-                    addressLine2: "",
-                    city: "",
-                    region: "",
-                  },
-                  phoneNumber: "",
-                  phoneNumberCountryCode: "",
-                }}
-                validationSchema={validationSchema}
-                onSubmit={(
-                  values: FormikValues,
-                  formikHelpers: FormikHelpers<FormikValues>,
-                ) => {
-                  console.warn("Delivery address values submitted:", values)
+            <Box data-testid="OrderSummary" backgroundColor="mono0">
+              <Flex py={1} px={2} justifyContent="space-between">
+                <Link
+                  flex={0}
+                  href={`/artwork/${TEMP_DATA.artworkSlug}`}
+                  target="_blank"
+                >
+                  <Image
+                    mr={1}
+                    src={TEMP_DATA.artworkVersionImageResizedUrl}
+                    alt={TEMP_DATA.artworkVersionTitle || ""}
+                    width="40px"
+                  />
+                </Link>
+                <Box
```

Comment:
#non-blocking - since we're doing something greenfield, might as well -- can we add some breathing room between components? Not everything needs a line break, but it makes things much more readable across time if stuff is separated a bit where it makes sense